src/reducers/users.tsx

```diff
@@ -0,0 +1,109 @@
+import invariant from 'invariant';
+import { Reducer } from 'redux';
+import { ActionType, createAction, getType } from 'typesafe-actions';
+
+type UserId = number;
+
+export type User = {
```

Comment:
This is only a subset of the [user response](https://addons-server.readthedocs.io/en/latest/topics/api/accounts.html#account) which is fine but I think you should follow our typical pattern for this where `User` is the internal representation and `ExternalUser` is a representation of what the API will respond with. The action functions will except `ExternalUser` and the reducer would store it as `createInternalUser(payload.user)`.

Reply:
I was trying to keep it simple by not even referencing any of the data we don't need from the API, but sure, I can go the `ExternalUser` approach.